CHANGELOG.md

```diff
@@ -2,8 +2,12 @@
 
 ## Unreleased
 - Add support for fetching address reputation
-  - Add `reputation` method to `Address` to fetch the reputation of the address. 
+  - Add `reputation` method to `Address` to fetch the reputation of the address
 - Add `networkId` to `WalletData` so that it is saved with the seed data and surfaced via the export function
+- Add ability to import external wallets into CDP via a BIP-39 mnemonic phrase, as a 1-of-1 wallet
+- Add ability to import WalletData files exported by the Python CDP SDK
+- Deprecate `Wallet.loadSeed()` method in favor of `Wallet.loadSeedFromFile()`
+- Deprecate `Wallet.saveSeed()` method in favor of `Wallet.saveSeedToFile()`
```

Comment:
We'll move these around as we prep the release, but we'll have a separate heading for "Address" and "Deprecated"